### PR TITLE
Add basic queries for bibtex

### PR DESCRIPTION
**Pull request by clason**
What it says on the tin: a first attempt at adding queries for bibtex:

* highlights (can be improved, suggestions for better captures appreciated)
* indents
* folds

@theHamsta

### CODE REVIEW

File: queries/bibtex/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+[
+  (brace_word)
+  (quote_word)
+] @string
+
+[
+  (key_brace)
+  (key_paren)
+] @attribute
```
Comment (theHamsta) [inline queries/bibtex/highlights.scm:37]: `attribute` has the moment the meaning of `code annotation`. We should probably change that... memo to myself. I would leave it that way.

Comment (clason) [reply queries/bibtex/highlights.scm:37]: yeah, I don't know the proper highlight groups for tree-sitter, I'm afraid. That's why the checks are failing 😞 

I'd  be grateful for suggestions for appropriate groups!

File: queries/bibtex/highlights.scm [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+(command) @function.builtin
+
+(number) @number
+
+(field
+  name: (identifier) @variable.builtin)
+
+(token
+  (identifier) @variable.parameter)
```
Comment (theHamsta) [inline queries/bibtex/highlights.scm:27]: This is `@parameter` for us. Should we add `@variable.parameter` as a comp mapping (needs to be added to hl_map in highlight.lua and contributing).

File: queries/bibtex/highlights.scm (hunk 1 above)
Comment (theHamsta) [reply queries/bibtex/highlights.scm:37]: no, attribute exists. The failure is due to `@variable.parameter`

Comment (clason) [reply queries/bibtex/highlights.scm:37]: I mean "the failure is due to the fact that I don't know what I'm doing, generally" :]

File: queries/bibtex/indents.scm [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[
+ (entry)
+] @indent
+
+[
+ "{"
+ "}"
+] @branch
+
+(comment) @ignore
```
Comment (theHamsta) [inline queries/bibtex/indents.scm:10]: so you could add yourself as maintainer for bibtex? Or we add an auto-import feature

Comment (clason) [reply queries/bibtex/indents.scm:10]: "I would prefer not to" :)

Comment (clason) [reply queries/bibtex/indents.scm:10]: (see above: not really knowing what I'm doing)

Comment (clason) [reply queries/bibtex/indents.scm:10]: the indents and folds were easy enough, though -- those I can manage :)

Comment (clason) [reply queries/bibtex/indents.scm:10]: But on a more serious note, I don't think we need an auto-import here -- the highlights queries need tuning, and the other ones are not from upstream.

**Review by theHamsta — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -134,7 +134,7 @@ We are looking for maintainers to add more parsers and to write query files for
 <!--parserinfo-->
 - [x] [bash](https://github.com/tree-sitter/tree-sitter-bash) (maintained by @TravonteD)
 - [x] [beancount](https://github.com/polarmutex/tree-sitter-beancount) (maintained by @polarmutex)
-- [ ] [bibtex](https://github.com/latex-lsp/tree-sitter-bibtex)
+- [x] [bibtex](https://github.com/latex-lsp/tree-sitter-bibtex)
```
Comment (theHamsta) [inline README.md:137]: this is automatically edited. the cross will only be added if there is a maintainer.

Comment (clason) [reply README.md]: Sorry, I was going by the text before the list.

(Which I wrote, so...)

**Review by vigoux — APPROVED** [review]
Great! Thanks!

**Comment by theHamsta** [comment]
Looks really nice!

**Pull request merged**